### PR TITLE
Add Date/Calendar templates for java language

### PR DESCRIPTION
**Pull request by cassiuscai**
- [Templates for the `Date/Calendar` API](https://github.com/cassiuscai/postfix-templates/blob/master/documents/date-operation.md)

### CODE REVIEW

File: templates/webTemplateFiles.yaml [hunk 1]
```diff
@@ -1,3 +1,31 @@
+- lang: java
+  id: cassiuscai.date-operation
+  url: https://raw.githubusercontent.com/cassiuscai/postfix-templates/master/templates/date-operation.postfixTemplates
```
Comment (xylo) [inline templates/webTemplateFiles.yaml]: I like these templates.  They are well-designed and limited to commons-lang.

File: templates/webTemplateFiles.yaml [hunk 2]
```diff
@@ -1,3 +1,31 @@
+- lang: java
+  id: cassiuscai.date-operation
```
Comment (xylo) [inline templates/webTemplateFiles.yaml]: I would probably name the file `cassiuscai.commons-lang.date`.

Comment (cassiuscai) [reply templates/webTemplateFiles.yaml]: Thanks for your advice, I'll change that.

**Comment by cassiuscai** [comment]
web templates updated

**Comment by xylo** [comment]
OK. Thanks.

**Pull request merged**